dashboard/assets/packages/report/package.json

```diff
@@ -36,7 +36,7 @@
     "eslint-plugin-prettier": "^5.0.1",
     "rollup-plugin-visualizer": "^5.9.2",
     "sass": "^1.65.1",
-    "vite": "^4.4.11",
+    "vite": "^4.5.14",
```

Comment:
`vite` also contains multiple CVEs as reported in:

https://github.com/advisories/GHSA-vg6x-rcgg-rjx6
https://github.com/advisories/GHSA-92r3-m2mg-pj97
https://github.com/advisories/GHSA-356w-63v5-8wf4
https://github.com/advisories/GHSA-859w-5945-r5v3
https://github.com/advisories/GHSA-8jhw-289h-jh2g
https://github.com/advisories/GHSA-9cwx-2883-4wfx
https://github.com/advisories/GHSA-x574-m823-4x7w
https://github.com/advisories/GHSA-4r4m-qw57-chr8
https://github.com/advisories/GHSA-xcj6-pq6g-qj4x